Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -30,6 +30,8 @@ public class FocusProvider : BaseCoreSystem, IMixedRealityFocusProvider
         private readonly Dictionary<uint, IMixedRealityPointerMediator> pointerMediators = new Dictionary<uint, IMixedRealityPointerMediator>();
         private PointerHitResult hitResult3d = new PointerHitResult();
         private PointerHitResult hitResultUi = new PointerHitResult();
+        private Collider[] colliderQueryBuffer;
+        private int sceneQueryBufferSize = 64;
```

Comment:
#4942 adds a configurable PokePointer property for this. The FocusProvider should probably use the PointerProfile like it does for other settings.

Reply:
I think all the changes in this file are no longer necessary, actually. Addison and I discussed a much simpler solution. Addison is on pat leave also. I'll take over the PR and just provide the two fixes in basecontrollerpointer and sphere pointer.